docs/source/admin/capacity-planning.md

```diff
@@ -0,0 +1,246 @@
+
+On the other hand, if you are using ephemeral resources,
+such as node pools in Kubernetes,
+changing resource types costs close to nothing
+because nodes can automatically be added or removed as needed.
+
+Take that cost into account when you are picking how much memory or cpu to allocate to users.
+
+Static resources (like [the-littlest-jupyterhub][]) provide for more **stable, predictable costs**,
+but elastic resources (like [zero-to-jupyterhub][]) tend to provide **lower overall costs**
+(especially when deployed with monitoring allowing cost optimizations over time),
+but which are **less predictable**.
+
+[the-littlest-jupyterhub]: https://the-littlest-jupyterhub.readthedocs.io
+
+[zero-to-jupyterhub]: https://zero-to-jupyterhub.readthedocs.io
+
+(limits-requests)=
+
+### Limit vs Request
+
+Many scheduling tools like Kubernetes have two separate ways of allocating resources to users.
+A **Request** or **Reservation** describes how much resources are _set aside_ for each user.
+Often, this doesn't have any practical effect other than deciding when a given machine is considered 'full'.
+If you are using expandable resources like an autoscaling Kubernetes cluster,
+'requesting' more resources than fit on currently running nodes is when a new node is launched and added to the pool (a cluster **scale-up event**).
+If you are running on a single VM, this describes how many users you can run at the same time, full stop.
+
+A **Limit**, on the other hand, actually enforces a limit to how much resources any given user can consume.
+We'll see more information on what happens when users try to exceed their limits [below](oversubscription).
+
+In the strictest, safest case, you can have these two numbers be the same.
+That means that each user is _limited_ to fit within the resources allocated to it.
+This avoids **[oversubscription](oversubscription)** of resources (allowing use of more than you have available),
+at the expense (in a literal, this-costs-money sense) of reserving lots of usually-idle capacity.
+
+But when deploying JupyterHub,
+you will likely find that a relatively small fraction of users use lots more resources than others,
+making oversubscription attractive (to a point).
+
+Having a gap between the request and the limit means you can fit a number of _typical_ users on a node (based on the request),
+but still limit how much a runaway user can gobble up for themselves.
+
+(oversubscription)=
+
+### Oversubscribed CPU is okay, running out of memory is bad
+
+An important consideration when assigning resources to users is:
+
+> What happens when users need more than I've given them?
+
+A good summary to keep in mind: **when tasks don't get enough CPU, things are slow.
+When they don't get enough memory, things are broken.**
+Which means it's very important that users have enough memory,
+but much less important that they always have exclusive access to all the CPU they can use.
```

Comment:
```suggestion
An important consideration when assigning resources to users is: **What happens when users need more than I've given them?**

A good summary to keep in mind:

> When tasks don't get enough CPU, things are slow.
  When they don't get enough memory, things are broken.

This means it's **very important that users have enough memory**,
but much less important that they always have exclusive access to all the CPU they can use.
```

This highlights the "important part" a bit more clearly

Reply:
Yeah, I was surprised by the lack of block quote formatting. I think I have a bug report for our sphinx theme on the way.